contrib/chinese/packages.el

```diff
@@ -14,13 +14,17 @@
 ;; which require an initialization must be listed explicitly in the list.
 (setq chinese-packages
     '(
-      youdao-dictionary
-      chinese-pyim
       find-by-pinyin-dired
       ace-pinyin
-      chinese-wbim
       ))
 
+(if chinese-enable-youdao-dict
+  (add-to-list 'chinese-packages 'youdao-dictionary))
```

Comment:
Heres an example: https://github.com/syl20bnr/spacemacs/blob/8dadfc1494544bb152e80c2a436e43bc3713b389/contrib/!tools/dash/packages.el#L5

The git layer also used to do this depending on the value of `git-gutter-use-fringe` before the switch to `diff-hl`. If you use `SPC h d f <function>` for `push` and `add-to-list`, one of the docstrings will recommend using `push` for most use-cases.